lib/scala/pkg/src/test/scala/org/enso/pkg/ConfigSpec.scala

```diff
@@ -36,7 +36,8 @@ class ConfigSpec
         services             = List(),
         jvm                  = None
       )
-      val deserialized = Config.fromYaml(config.toYaml).get
```

Comment:
Were these tests not executed in the CI? I'm still trying to understand what #11805 could break and why that's bad? E.g. why not fix the test?

Reply:
I saw the `LibrariesTest` failure before locally, but never on the CI for my PRs, so I assumed it is some build mismatch problem.

- why doesn't it fail the CI for my PRs?
- is `language-server/test` executed?
- if so, does it succeeds because CI for the PRs sets version to 2025.x.y?
- can't we also set a version to 2025.x.y for the nightly checks then?